backend/src/api/routes/github/index.routes.ts

```
@@ -0,0 +1,27 @@
+const router = Router();
+const gitHubService = GitHubService.getInstance();
+
+router.use(verifyAdmin);
+
+/**
+ * GET /api/github/repository
+ * Returns cached GitHub repository metadata.
+ */
+router.get(
+  '/repository',
+  async (_req: AuthRequest, res: Response, next: NextFunction) => {
```

ESLint

[error] Replace `⏎··'/repository',⏎·` with `'/repository',`

(prettier/prettier)